frontend/public/components/app.jsx

```diff
@@ -229,6 +230,7 @@ render(
   <Provider store={store}>
     <Router history={history} basename={window.SERVER_FLAGS.basePath}>
       <Switch>
+        <Route path="/vm-console" component={CloudVMConsole} />
```

Comment:
IMHO our route table is a better place for a route we use, see for example:
https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/plugin.tsx#L158

Reply:
Mounting the route here will mount the layout aswell, we want to render the console alone

Reply:
@vojtechszocs @spadgett 

Are we okay with rendering a kubevirt component here? Are we going to add an extension point for this in the future?

Reply:
> Mounting the route here will mount the layout aswell, we want to render the console alone

With #5645 a plugin can add new page & route that is rendered outside the standard Console `<App>` layout.

I think the `CloudVMConsole` page fits this use case, so we shouldn't need to modify the core `app.jsx`.